resources/lang/ru/twitch.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Russian translation strings for generic Twitch-related errors.
+ * done by paws of kriper1111
+ */
+return [
+    /**
+     * Related to chat
+     */
+    'no_chat_rules' => 'На канале :channel не установлено никаких правил.',
+    'error_occurred_chat_clusters' => 'Произошла ошибка при получении кластеров чата.',
+    'error_retrieving_chat_users' => 'Произошла ошибка при получении пользователей канала channel: ',
+    'empty_chat_user_list' => 'Список пользователей пуст.',
+    'channel_missing_subemotes' => 'На этом канале нету смайлов сабскрайберов.',
+
+    /**
+     * Related to followers
+     */
+    'cannot_follow_self' => 'Пользователь не может фолловить себя.',
+    'error_followers' => 'Произошла ошибка при получении списка ваших фолловеров.',
+    'follow_not_found' => ':user не зафолловлен на :channel',
+    'no_followers' => 'У вас нету подписчиков :(',
+    'invalid_api_data' => 'Twitch API вернул неправильные данные.',
+    'unable_get_following' => 'Невозможно получить данные о фолловерах указанного пользователя.',
+    'end_following_list' => 'Конец списка фоллова.',
+
+    /**
+     * Help articles
+     */
+    'help_articles' => 'Справка',
+    'help_available_list' => 'Список доступных страниц справки по запросу: :url',
+    'help_no_results' => 'Результаты не найдены.',
+
+    /**
+     * Media (highlights, VODs, uploads)
+     */
+    'no_highlights' => 'На канале :channel нету сохранённых хайлайтов.',
+    'no_uploads' => 'На канале :channel нету загруженных видео.',
+    'no_vods' => 'На канале :channel нету доступных VOD'ов.',
```

Comment:
You'll have to escape this, because of the single quote:

`'На канале :channel нету доступных VOD'ов.'` => `'На канале :channel нету доступных VOD\'ов.'`